### ISSUE TITLE
WET4 For HTML EMail Template Issues

### ISSUE THREAD

**Issue by massierick** [wet-boew/wet-boew#8217]
Hi,
I've been trying to use WET4 as an email template, but have been running into issues with how the page displays. Most of the CSS in WET4 appears to be non-functional in the email for some reason. 
I'm using Outlook 2010, opening a new email, and using the Insert > Attach File >Insert as text function to load the HTML into the email. 
For the WET4 files, I've been linking directly to the files hosted on the INAC (AANDC) website, since there doesn't appear to be a generic CDN for the WET4 files. If you check the code below, you'll see that the header links for CSS and JS point to the AANDC server. 

Long story short - I've been trying to get this to work in email, and it just won't work for me. Any help would be much appreciated!

View in browser:
![wet4-browser](https://user-images.githubusercontent.com/15625062/33397120-7fec89e2-d4ff-11e7-832b-2e130f17f948.JPG)

View in email:
![wet4-email](https://user-images.githubusercontent.com/15625062/33397132-86fcaf6e-d4ff-11e7-8258-8d599715ff7b.JPG)

Code:

[newsletter-test2.txt](https://github.com/wet-boew/wet-boew/files/1515390/newsletter-test2.txt)

**Comment by RobJohnston** [comment]
Maybe it's just the bizarre network at IRCC, but when I load the following files, I get a network error:


> A communication error occurred: "Operation timed out"
> --
> The Web Server may be down, too busy, or experiencing other problems preventing it from responding to requests. You may wish to try again at a later time.

- http://www.aadnc-aandc.gc.ca/DAM/PresentationDAM/STAGING/WET/WET4.0.25-USABILITY/theme-gcwu-fegc/css/wet-boew.min.css
- 
- http://www.aadnc-aandc.gc.ca/DAM/PresentationDAM/STAGING/WET/WET4.0.25-USABILITY/theme-gcwu-fegc/css/theme.min.css

**Comment by massierick** [comment]
Thanks for checking it out Rob. The files load fine for me, perhaps you're right about the IRCC network issue. 

Is there a CDN for WET4? I've used them for jquery, and I'm basically trying to emulate one by pulling from aandc.gc.ca , but perhaps it would be safer to try it from a stable CDN if it actually exists.

**Comment by nschonni** [comment]
Email has limited support for CSS https://www.campaignmonitor.com/css/
WET also relies alot on JavaScript for element enhancement so I don't think it would play that well with email templates

**Comment by kodecount** [comment]
You can try something like:
```
<!DOCTYPE html>
<html>
  <head>
      <meta charset='UTF-8'>
      <title></title>
      <style>
      .body{max-width:1100px;margin:auto;padding:30px;border:1px solid #eee;box-shadow:0 0 10px rgba(0, 0, 0, .15);font-size:16px;line-height:24px;font-family:'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif;color:#555;background: white;}
      .body table{width:100%;line-height:inherit;text-align:left;}
      .body table td{padding:5px;vertical-align:top;}
      .body table tr.logo td{padding-bottom:20px;}
      .body table tr.logo td a {display: block;text-decoration: none;}
      .body table tr.logo td.title{font-size:45px;line-height:45px;color:#333;}
      .body table tr.header td{padding-bottom:40px;}
      .body table tr.bxTitle td{background: #114f7a;background: -webkit-gradient(linear,left top,left bottom,from(#176ca7),to(#114f7a));background: -webkit-linear-gradient(#176ca7,#114f7a);background: linear-gradient(#176ca7,#114f7a);border-bottom:1px solid #114f7a;font-weight:bold;color: white;}
      .body table tr.bxInfo td{border: 2px solid #114f7a;border-top: none;}
      .body table tr.bxInfo td:nth-child(2){border: 2px solid #114f7a;border-top: none;border-left: none;}
      .wb-invisible, .wb-inv, .wb-show-onfocus, .wb-sl, #wb-lng h2, #wb-glb-mn h2, #wb-srch h2, #wb-srch label, #wb-sm h2, #wb-bc h2, #wb-sec h2, #wb-info h2, #mb-pnl h3, .wb-calevt-cal .cal-days td ul, .wb-fnote dt, #mb-pnl .srch-pnl label, #wb-lng .curr span {clip: rect(1px, 1px, 1px, 1px);height: 1px;margin: 0;overflow: hidden;position: absolute;width: 1px; }
      .wrapImage{display: inline-flex;width: 100%;}
      .wrapImage p{width: 100%;}
      .text-right{text-align: right !important;}
      body{background: #f9f9f9;}
      small{font-size: 80%;}
      @media only screen and (max-width: 600px) {
        .body table tr.logo td{width:100%;display:block;text-align:center;}
        .body table tr.header td{width:100%;display:block;text-align:center;}
      }
      </style>
    </head>
<body>
  <div class='body'>
    <table  cellpadding='0' cellspacing='0'>
      <tr class='logo'>
        <td><a href="http://wet-boew.github.io/themes-dist/GCWeb/index-en.html"><img type="image/svg+xml" width="33%" tabindex="-1" src="http://wet-boew.github.io/themes-dist/GCWeb/assets/sig-blk-en.png"></object><span class="wb-inv"> Government of Canada / <span lang="fr">Gouvernement du Canada</span></span></a></td>
      </tr>
      <tr class='header'>
        <td><h1>2017 Newsletter</h1></td>
      </tr>
      <tr class='bxTitle'>
        <td>Article 1: Testing email templates</td>
      </tr>
      <tr class='bxInfo'>
        <td>
          <div class="wrapImage">
            <p>
            This is just an example.<br/>
            This is just an example.
            </p>
            <div class="text-right">
              <img src="http://wet-boew.github.io/themes-dist/GCWeb/img/360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
            </div>
          </div>
        </td>
      </tr>
    </table>
  </div>
</body>
</html>
```
Example at http://kodecount.ca/Email_Template.html

**Comment by rlambert27** [comment]
You could pull WET files from the CDTS solution : http://www.gcpedia.gc.ca/wiki/Centrally_Deployed_Templates_Solution_(CDTS)

The root of any WET version (since v4.0.21) is located at : https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_26/ (replace v4_0_26 with the version you would like)

Or you could always grab the latest files from : https://www.canada.ca/etc/designs/canada/cdts/gcweb/rn/

**Comment by massierick** [comment]
Thanks everyone, I'll give these suggestions a shot and report back.
